quran/quran.com-frontend-next
# add 'rtl' attribute when setting the language to Arabic

### Summary

We are using `next-translate` to add localization. When switching to Arabic, we need to let the browser know that the text direction of the element's content is `rtl` this is done by setting the `dir` attribute on `main` element like this: 
`<main dir='rtl'>`

### Test Plan

1. Visit `{url}/ar/about` and we should find the content shown correctly from right to left since we are changing the language to Arabic.
2. Visit `{url}/about` and we should find the content shown correctly from left to right.


### Screenshots

| Before | After |
| ------ | ------ |
| <img width="1440" alt="Screen Shot 2021-07-15 at 09 57 24" src="https://user-images.githubusercontent.com/15169499/125721430-fef6834a-71f3-458c-b204-a573262b6c36.png"> | <img width="1440" alt="Screen Shot 2021-07-15 at 09 56 49" src="https://user-images.githubusercontent.com/15169499/125721426-2f415936-522c-4ee8-8cc3-f73f7297d20b.png"> |
